app/Console/UpdateUniqueLabels.php

```diff
@@ -130,7 +131,7 @@ private function updateLabel(Uuid $labelUuid, LabelName $labelName): void
             );
     }
 
-    private function getLabelUuid(array $labelAddedEvent): Uuid
+    private function getLabelUuid(array $labelAddedEvent): UuidInterface
```

Comment:
Nit: We can probably just use the `Uuid` class as return type here as we did before? We also use it everywhere else in this class instead of `UuidInterface`

Reply:
The issue is that the `Uuid::fromString` returns a `UuidInterface`